packages/@expo/config-plugins/src/android/BuildProperties.ts

```diff
@@ -72,6 +72,19 @@ export const withNewArchEnabledGradleProps = createBuildGradlePropsConfigPlugin<
   'withNewArchEnabledGradleProps'
 );
 
+/**
+ * A config-plugin to update `android/gradle.properties` from the `updates.useNativeDebug` in expo config
+ */
+export const withUpdatesNativeDebugGradleProps = createBuildGradlePropsConfigPlugin<ExpoConfig>(
+  [
+    {
+      propName: 'EX_UPDATES_NATIVE_DEBUG',
+      propValueGetter: (config) => (config?.updates?.useNativeDebug ?? false).toString(),
+    },
+  ],
+  'withUpdatesNativeDebugGradleProps'
+);
+
```

Comment:
leaning toward to put updates related plugins together at https://github.com/expo/expo/blob/main/packages/%40expo/config-plugins/src/android/Updates.ts.

also you've added `removePropWhenValueIsNull=true` on ios. would you like to have a consistent behavior on android?

Reply:
Moved to Updates.ts. Also reverted the `removePropWhenValueIsNull` change.